docs/glossary.rst

```diff
@@ -56,11 +62,14 @@ Glossary
       A placeholder representing a runtime or |Network|-specific value, at
       a particular byte offset or offsets. Identified by name.
 
-      Filled in by corresponding |LinkValue| by Truffle.
+      Filled in with corresponding |LinkValue| by Truffle.
 
    Link Value
       The value used to replace |LinkReference| placeholders.
 
+   Name
```

Comment:
I basically plagiarized this from the part in the data model...

Reply:
I don't think it's plagiarism to take words from a different section of the same document